cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,62 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+const faker = require("faker");
 
+describe('Settings page', () => {
   beforeEach(() => {
+    cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+      cy.login(user).then(() => user);
+    }).as('user');
 
+    cy.visit('#/settings')
   });
 
   it('should provide an ability to update username', () => {
-
+    const newUsername = faker.internet.userName();
+    cy.get('@user').then(user => {
+      cy.getByDataCy('username-settings')
+        .clear().type(newUsername);
+        cy.getByDataCy('update-settings').click();
+      cy.contains('.swal-modal', 'Update successful!').should('exist');
+    });
   });
 
   it('should provide an ability to update bio', () => {
+    const bio = faker.lorem.sentence();
 
+    cy.get('@user').then(user => {
+      cy.getByDataCy('bio-settings').type(bio);
+      cy.getByDataCy('update-settings').click();
+      cy.contains('.swal-modal', 'Update successful!').should('exist');
+    });
   });
 
   it('should provide an ability to update an email', () => {
-
+      const newEmail = faker.internet.email().toLowerCase();
+  
+      cy.get('@user').then(user => {
+        cy.getByDataCy('email-settings').clear().type(newEmail);
+        cy.getByDataCy('update-settings').click();
+        cy.contains('.swal-modal', 'Update successful!').should('exist');
+      });
   });
 
   it('should provide an ability to update password', () => {
+    const newPassword = 'ChangedPassword1!'
 
```

Comment:
Avoid using hardcoded passwords in tests. You could use faker to generate random passwords.